src/components/appNavigation/TabsView.js

```diff
@@ -85,12 +84,12 @@ const TabsView = (props: TabViewProps) => {
   return (
     <Appbar.Header dark>
       <TouchableOpacity onPress={goToRewards} style={{ marginLeft: '10px' }}>
-        <RewardSvg />
+        <Icon name="rewards" size={36} color="white" />
       </TouchableOpacity>
       <Appbar.Content />
       {config.market && (
         <TouchableOpacity onPress={goToMarketplace} style={styles.marketIconBackground}>
-          <MarketPlaceSvg />
+          <Icon name="marketplace" size={51} color="white" />
```

Comment:
are you sure about those font sizes? 51? 36? seems strange to me
attach image please

Reply:
@sirpy 
Screenshots:

| | |
| --- | --- |
| <img width="476" alt="1" src="https://user-images.githubusercontent.com/49862004/68839550-9a96f900-06c9-11ea-8d5c-ff8929f45e14.png"> | <img width="336" alt="2" src="https://user-images.githubusercontent.com/49862004/68839565-a08cda00-06c9-11ea-9c83-34cee462ef1e.png"> |